src/browser/telemetry.js

```diff
@@ -354,7 +354,9 @@ Instrumenter.prototype.instrumentNetwork = function() {
           }
         }
         self.captureNetwork(metadata, 'fetch', undefined);
-        metadata.stack = (new Error()).stack;
+        if (self.trackHttpErrors()) {
+          metadata.stack = (new Error()).stack;
```

Comment:
Should we send `stack = null` here or is ok not sending the `stack` property at all?

Reply:
I think not setting at all would be the preference.

Reply:
:+1: